README.md

```diff
@@ -34,11 +34,13 @@ The following are the environment variables for a backup:
 
 __You should consider the [use of `--env-file=`](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables-e-env-env-file), [docker secrets](https://docs.docker.com/engine/swarm/secrets/) to keep your secrets out of your shell history__
 
+* `TZ`: set the timezone used for backup timestamps, for example `TZ=Europe/Vienna`, defaults to `UTC`
 * `DB_SERVER`: hostname to connect to database. Required.
 * `DB_PORT`: port to use to connect to database. Optional, defaults to `3306`
 * `DB_USER`: username for the database
 * `DB_PASS`: password for the database
-* `DB_NAMES`: names of databases to dump; defaults to all databases in the database server
+* `DB_NAMES`: names of databases to dump (seperated by space); defaults to all databases in the database server
+* `DB_NAMES_EXCLUDE`: names of databases (seperated by space) to exclude from the dump; `information_schema`. `performance_schema`, `sys` and `mysql` are excluded by default. This only applies if `DB_DUMP_BY_SCHEMA` is set to `true`. For example, if you set `DB_NAMES_EXCLUDE=database1 db2` and `DB_DUMP_BY_SCHEMA=true` then theese two databases will not be dumped by mysqldump.
```

Comment:
* "separated" (replaces "seperated")
* "these" (replaces "theese")
* `DB_NAMES_EXCLUDE="database1 db2"` (add the quotes please)